src/main/java/com/plaid/client/request/LinkTokenCreateRequest.java

```diff
@@ -0,0 +1,172 @@
+    }
+
+    public User withPhoneNumber(String phoneNumber) {
+      this.phoneNumber = phoneNumber;
+      return this;
+    }
+
+    public User withEmailAddress(String emailAddress) {
+      this.emailAddress = emailAddress;
+      return this;
+    }
+
+    public User withPhoneNumberVerifiedTime(Date phoneNumberVerifiedTime) {
+      if (phoneNumberVerifiedTime == null) {
+        this.phoneNumberVerifiedTime = Optional.empty();
+      } else {
+        this.phoneNumberVerifiedTime = Optional.of(phoneNumberVerifiedTime);
+      }
+      return this;
+    }
+
+    public User withEmailAddressVerifiedTime(Date emailAddressVerifiedTime) {
+      if (emailAddressVerifiedTime == null) {
+        this.emailAddressVerifiedTime = Optional.empty();
+      } else {
+        this.emailAddressVerifiedTime = Optional.of(emailAddressVerifiedTime);
+      }
+      return this;
+    }
+  }
+
+  public static class CrossAppItemAdd {
+    private String targetApplicationToken;
+    private String foreignId;
+
+    public CrossAppItemAdd(String targetApplicationToken, String foreignId) {
+      this.targetApplicationToken = targetApplicationToken;
+      this.foreignId = foreignId;
+    }
+  }
+
+  public LinkTokenCreateRequest withUser(User user) {
+    this.user = user;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withClientName(String clientName) {
+    this.clientName = clientName;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withProducts(List<String> products) {
+    this.products = products;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withCountryCodes(List<String> countryCodes) {
+    this.countryCodes = countryCodes;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withLanguage(String language) {
+    this.language = language;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withWebhook(String webhook) {
+    this.webhook = webhook;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withAccessToken(String accessToken) {
+    this.accessToken = accessToken;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withLinkCustomizationName(String linkCustomizationName) {
+    this.linkCustomizationName = linkCustomizationName;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withRedirectUri(String redirectUri) {
+    this.redirectUri = redirectUri;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withAndroidPackageName(String androidPackageName) {
+    this.androidPackageName = androidPackageName;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withAccountFilters(Map<String, SubtypeFilters> accountFilters) {
+    this.accountFilters = accountFilters;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withCrossAppItemAdd(CrossAppItemAdd crossAppItemAdd) {
+    this.crossAppItemAdd = crossAppItemAdd;
+    return this;
+  }
+
+  public LinkTokenCreateRequest withPaymentInitiation(String paymentInitiation) {
+    this.paymentInitiation = paymentInitiation;
```

Comment:
The shape of payment_initiation should be:
`{ payment_initiation: { payment_id: `payment_id` } }`

Reply:
where `payment_id` is required if `payment_initiation` is configured